common/company_types/00_companies_africa.txt

```diff
@@ -115,7 +115,8 @@ company_suez_company = {
 	
 	prosperity_modifier = {
 		country_trade_route_competitiveness_mult = 0.25
-		country_tax_income_add = 50000
+		country_minting_mult = 0.20
```

Comment:
is this a buff or a debuff? Because it reads like a debuff, but late game this is a buff.

Reply:
It's neither, its a rework of the company bonus so that you get less free money, but more profit tied to your GDP and tech. It benefits countries with larger economies more and ones with smaller economies less, though all eventually tend to become more profitable over time.